django-transparent/docker-compose.django-transparent.yml

```diff
@@ -6,7 +6,7 @@ services:
     # - ./.acrakeys/acra-server/${ACRA_CLIENT_ID}_storage.pub
     # - ./.acrakeys/acra-server/${ACRA_CLIENT_ID}_storage_sym
     acra-keymaker_client:
-        image: "cossacklabs/acra-keymaker:${ACRA_DOCKER_IMAGE_TAG:-0.93.0}"
+        image: "cossacklabs/acra-keymaker:${ACRA_DOCKER_IMAGE_TAG:-0.92.0}"
```

Comment:
actually we should leave 0.93.0 )

Reply:
Is it actual change? I had it, but fixed and dont see it in changed files.

Reply:
hm, looks like a bug in github. because on `files` tab everything is okay
![image](https://user-images.githubusercontent.com/6237700/164709466-ac7ab080-29dc-4d5e-af39-4cbbb755d643.png)